src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraDeploymentSpec.kt

```diff
@@ -9,9 +9,10 @@ enum class TemplateType {
     deploy, development, localTemplate, template, build
 }
 
-enum class ApplicationPlatform(val baseImageName: String, val baseImageVersion: String) {
-    java("flange", "8"),
-    web("wrench", "0")
+
```

Comment:
Enum is probably not the correct choice here anymore. Also i can see that already we need to be able to tell some metadata about the different containers here.

Right now the service is automatically connected to 8080 but that might not always be the case